pkg/cmd/kafka/consumergroup/describe/describe.go

```diff
@@ -93,6 +94,10 @@ func NewDescribeConsumerGroupCommand(f *factory.Factory) *cobra.Command {
 		return cmdutil.FilterValidConsumerGroupIDs(f, toComplete)
 	})
 
+	_ = cmd.RegisterFlagCompletionFunc("output", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
```

Comment:
Good work but this is being duplicated in every file and they are the same. Could you extract this into the flagutil package?